app/build.gradle

```diff
@@ -65,6 +60,11 @@ android {
             matchingFallbacks = ['debug']
             initWith buildTypes.debug
         }
+        nightly {
```

Comment:
Can this be added for all but debug flavours? 

Consequently, is there a need for a *nightly* flavour?

Reply:
The nightly variant is there because I didn't want to sign the Apk there is no need for it, for signing release builds you need to add the base64 encoded signing key and password as 2 separate github secret entries

Reply:
Does it make sense to build `debug` *every* nightly, then?

Reply:
The nightly exists because debug builds are not minified which make them bigger and less performant the bigger Apk size is important because Telegram Bot API does not allow files more than 50MB to be uploaded and the rethink Apk is more than 50 MB without minification.

Reply:
gotcha; you can perhaps add this reason as a comment in the gradle file?

Also, does adding `nightly` affect `fdroid` builds? Currenly, F-Droid builds the `fdroidFull` flavour for Rethink.